server/service/service_errors.go

```diff
@@ -21,3 +21,29 @@ func (a alreadyExistsError) Error() string {
 func (a alreadyExistsError) IsExists() bool {
 	return true
 }
+
+// ssoErrCode defines a code for the type of SSO error that occurred.
```

Comment:
Would it be appropriate to add something to the comment indicating how the `ssoErrCode` is intended to be used (similar to the explanation from the changes file above)?